### PR TITLE
Feature/contact us form

### PR DESCRIPTION
**Pull request by rohithmsr**
## Issue that this pull request solves

 Closes: #499 

### Brief description of what is fixed or changed

Contact Us Model to store the response of the users is created and as soon as the user sends the response, an email is sent
to the admins following a template mentioned in utility folder.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
------------

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: backend/app/models/ContactUs.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      type: String,
+      required: true,
+      min: [8, 'Message length must be at least 8 characters long'],
+      trim: true,
+    },
+  },
+  { timestamps: { createdAt: 'createdAt', updatedAt: 'updatedAt' } }
+);
+
+module.exports = mongoose.model('ContactUsUsers', contactUsUserSchema);
```
Comment (jackfrost13) [inline backend/app/models/ContactUs.js:34]: This should be just contactUs. Applicable for everywhere

File: backend/app/routes/contactus/@validationSchema/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const Joi = require('joi');
+
+const postResponseSchema = Joi.object({
+  name: Joi.string()
```
Comment (jackfrost13) [inline backend/app/routes/contactus/@validationSchema/index.js:4]: for this everything is requried

File: backend/app/routes/contactus/getAdmins.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
```
Comment (jackfrost13) [inline backend/app/routes/contactus/getAdmins.js:5]: We already have a function for this, please you that

File: backend/app/routes/contactus/@validationSchema/index.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const Joi = require('joi');
+
+const postResponseSchema = Joi.object({
```
Comment (jackfrost13) [inline backend/app/routes/contactus/@validationSchema/index.js:3]: name should be postContactUsValidationSchema

File: backend/app/routes/contactus/sendEmails.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+const { ErrorHandler } = require('../../../helpers/error');
+const { sendEmail } = require('../../../helpers/emailService');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, emails, next) => {
+  const data = { ...req.body };
+
+  try {
+    emails.map(async (item) => {
```
Comment (jackfrost13) [inline backend/app/routes/contactus/sendEmails.js:9]: you can directly call the send mail with a list of emails

File: backend/app/routes/contactus/postResponse.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    return next(error);
+  }
+
+  const getAdminEmails = await getAdmins(req, res, next);
+
+  await sendEmails(req, res, getAdminEmails, next);
+
+  res.status(200).send({
+    message: 'Your request has been submitted!',
+    your_response: response,
```
Comment (jackfrost13) [inline backend/app/routes/contactus/postResponse.js:30]: this response shouldn't be sent. You are not filtering the response

File: .gitignore [hunk 7]
```diff
@@ -1,3 +1,4 @@
 backend/node_modules/
 .vscode
 backend/package-lock.json
+backend/.env
```
Comment (jackfrost13) [inline .gitignore:4]: this shouldn't be ignored for now

**Review by jackfrost13 — CHANGES_REQUESTED** [review]
Please refactor the required things, Also you need to provide the api testing file(mocha-chai) and a brief backend documentation for the module

**Comment by jackfrost13** [comment]
@rohithmsr Are you working on this?

**Comment by rohithmsr** [comment]
No

**Pull request closed without merging**